Plugin~/WebRTCPlugin/UnityVideoEncoderFactory.cpp

```diff
@@ -32,13 +32,7 @@ namespace webrtc
 
     std::vector<webrtc::SdpVideoFormat> UnityVideoEncoderFactory::GetHardwareEncoderFormats() const
     {
-        const absl::optional<std::string> profileLevelId =
-            webrtc::H264::ProfileLevelIdToString(webrtc::H264::ProfileLevelId(webrtc::H264::kProfileConstrainedBaseline, webrtc::H264::kLevel5_1));
-        return { webrtc::SdpVideoFormat(
-            cricket::kH264CodecName,
-            { {cricket::kH264FmtpProfileLevelId, *profileLevelId},
-              {cricket::kH264FmtpLevelAsymmetryAllowed, "1"},
-              {cricket::kH264FmtpPacketizationMode, "1"} }) };
+        return { webrtc::CreateH264Format(webrtc::H264::kProfileConstrainedBaseline, webrtc::H264::kLevel5_1, "1") };
```

Comment:
This Implement same
https://webrtc.googlesource.com/src/+/refs/heads/master/modules/video_coding/codecs/h264/h264.cc#48